pandas/tests/indexing/test_datetime.py

```diff
@@ -10,6 +10,21 @@
 
 
 class TestDatetimeIndex:
+    def test_datetimeindex_transpose_empty_df(self):
+        """
```

Comment:
Did you post the link on purpose instead of only the reference like GH#xxx ? If yes, ok for me

Reply:
Yes, on (some) terminal emulators, clicking on the link will auto open them.
Also when seeing the code via a browser, it's easier to copy it, and paste it in the URL bar instead of: copying the number and then searching it in the issues tab.

---
TL;DR

This is for convenience, but not a deal breaker. I can have it as `GH#XXXX` if it's a deal breaker:)

Reply:
No strong opinion, would prefer consistency, but if you have a strong optinion this is fine with me